### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by tczaude**
I recreate polish lng this is translation for install

### CODE REVIEW

**Comment by d00p** [comment]
Nice, thanks, but what about the actual integration into the install? it won't be shown in the selection box if not specified, see class.FroxlorInstall.php

**Comment by tczaude** [comment]
I translate Yet main lng file if i finish. I add line in  class.FroxlorInstall.php on github 
Best regards

**Comment by tczaude** [comment]
Translate is done can by included in master 
Best regards

File: install/lib/class.FroxlorInstall.php [hunk 1]
```diff
@@ -66,7 +66,8 @@ class FroxlorInstall {
 	private $_languages = array(
 			'german' => 'Deutsch',
 			'english' => 'English',
-			'french' => 'Français'
+			'french' => 'Français',
+			'polish' => 'Polish'
```
Comment (d00p) [inline install/lib/class.FroxlorInstall.php]: the value (right side) of the array-item should be the language-name in the corresponding language, not english, in this case i guess "Polski"

**Comment by d00p** [comment]
In **lng/polish.lng.php** the line
```php
$lng['translator'] = 'translator';
```
should be your name, not the word "translator"

**Comment by tczaude** [comment]
Done :)

**Comment by HaibaneNero** [comment]
Just flying over the diff.... there is a tiny typo:
```Diff
--- a/lng/polish.lng.php
+++ b/lng/polish.lng.php
@@ -1418,7 +1418,7 @@ $lng['country']['KY'] = "Kajmany";
 $lng['country']['CF'] = "Central African Republic";
 $lng['country']['TD'] = "Czad";
 $lng['country']['CL'] = "Chile";
-$lng['country']['CN "] =" Chiny ";
+$lng['country']['CN'] = "Chiny";
 $lng['country']['CX'] = "Christmas Island";
 $lng['country']['CC'] = "Cocos (Keeling)";
 $lng['country']['CO'] = "Kolumbia";
```

**Comment by tczaude** [comment]
Thanks

**Comment by HaibaneNero** [comment]
@tczaude You are pretty fast, i couldn't find the error anymore on further inspection, not having realized you already fixed it. Deleted my comment assuming i confused something. Nice work^^ .... and friggin fast  :/

**Comment by tczaude** [comment]
All is should be done.

**Pull request closed without merging**